### PR TITLE
Added medication statement question

### PR DESCRIPTION
**Pull request by khavinshankar**
## Proposed Changes

- Added medication statement question

Screenshot:
image

@ohcnetwork/care-fe-code-reviewers

### CODE REVIEW

**Comment by bodhish** [comment]
Lets keep it simple? 
image


- Keep doasage as text
- Make soruce a dropdown. Default to patient
- Should we rename effective period to Started and Ended? Something simpler for users?

File: src/types/questionnaire/base.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export type Period = {
+  id?: string;
```
Comment (bodhish) [inline src/types/questionnaire/base.ts]: Does it have an ID 🤔 

Coudn't find one in FHIR spec https://build.fhir.org/datatypes.html#period

File: src/types/emr/medicationStatement.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  reason?: string;
+
+  medication: Code;
+  dosage?: string;
+  effective_period?: Period;
+
+  patient: string; // UUID
+  encounter: string; // UUID
+
+  information_source?: MedicationStatementInformationSourceType;
```
Comment (bodhish) [inline src/types/emr/medicationStatement.ts:36]: This should an enum!

File: src/types/emr/medicationStatement.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+export type MedicationStatementStatus =
+  (typeof MEDICATION_STATEMENT_STATUS)[number];
+
+export type MedicationStatement = {
+  readonly id: string;
+  status: MedicationStatementStatus;
+  reason?: string;
+
+  medication: Code;
+  dosage?: string;
```
Comment (bodhish) [inline src/types/emr/medicationStatement.ts]: Call it `dosage_text` so that we are clear that we storing the information as text

**Review by bodhish — CHANGES_REQUESTED** [review]
Lets also add a list of ongoing medications to the health profile page. `/facility/ID/patient/ID/health-profile`

**Comment by bodhish** [comment]
@khavinshankar ping me once you push the changes

**Comment by khavinshankar** [comment]
Updated UI:
image

**Pull request merged**